app/ui-sidenav/client/sidebarItem.html

```diff
@@ -56,6 +56,15 @@
 								<div class="sidebar-item__last-message {{#if showUnread }}{{#if lastMessageUnread }} sidebar-item__last-message--unread{{/if}}{{/if}}">
 									<span class="message-body--unstyled">{{{lastMessage}}}</span>
 								</div>
+								{{#if lastMessageSentByUser}}
```

Comment:
Please provide a screenshot of this UI change.

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/17441#issuecomment-671018517